src/components/collection.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { Box } from '@mui/material';
+import { useUser } from '@auth0/nextjs-auth0/client';
+
+const Collection = () => {
+
+  const { user, error, isLoading } = useUser();
+
+  // Make a loading component later to make this more interesting
+  if (isLoading) return <div>Loading...</div>
+  if (error) return <div>{error.message}</div>
+
+  // If logged in then display a welcome message otherwise prompt user to login
+  const renderCollection = () => {
+    if (user) {
+      return <h2>This is where the persons collection will display</h2>
+    } else {
+      return <h2>Please login to your account</h2>
+    }
+  }
+
+  return (  
+      <Box sx={{ width: "100vw", height: "100vh", backgroundColor: '#012835', pt:'64px', display:'flex', flexDirection: 'column'}}>
```

Comment:
All this custom inline styling is going to be a pain to maintain longterm but if the aim is to get stuff on the page and deal with that later then fine!

Reply:
Yeah really it was just so i could see roughly what it would look like